frontend/src/Stats.js

```diff
@@ -11,28 +14,52 @@ import {
   CartesianGrid,
   ResponsiveContainer,
 } from 'recharts'
+import theme from './theme'
 
 class Chart extends PureComponent {
   render() {
     return (
-      <div style={{ width: '100%', height: 300 }}>
-        <ResponsiveContainer>
-          <AreaChart
-            data={this.props.data}
-            margin={{
-              top: 0,
-              right: 0,
-              left: -25,
-              bottom: 0,
-            }}
-          >
-            <CartesianGrid strokeDasharray="2 2" />
-            <XAxis dataKey="date" />
-            <YAxis dataKey="total" />
-            <Area type="monotone" dataKey="total" stroke="#000" fill="#999" />
-          </AreaChart>
-        </ResponsiveContainer>
-      </div>
+      <I18n>
+        {({ i18n }) => (
+          <React.Fragment>
+            <H3 mt={6} textAlign="center">
+              <Trans>Reports similar to yours</Trans>
+            </H3>
+            <div style={{ width: '100%', height: 300 }}>
+              <ResponsiveContainer>
+                <AreaChart
+                  data={this.props.data}
+                  margin={{
+                    top: 0,
+                    right: 0,
+                    left: -20,
+                    bottom: 0,
+                  }}
+                >
+                  <CartesianGrid strokeDasharray="2 2" />
+                  <XAxis fontFamily={theme.fontSans} dataKey="date" />
+                  <YAxis
+                    dataKey="total"
+                    fontFamily={theme.fontSans}
+                    label={{
+                      value: i18n._(t`Reports per day`),
```

Comment:
Couldn't just use `<Trans>` here, instead used the approach suggested in https://lingui.js.org/tutorials/react-patterns.html#element-attributes-and-string-only-translations